corehq/apps/locations/fixtures.py

```diff
@@ -27,16 +29,35 @@ def __contains__(self, item):
         return item in self.by_id
 
 
-def should_sync_locations(last_sync, location_db):
+def fixture_last_modified(user):
```

Comment:
This one is in `locations/fixtures.py`, so it seems fine for it to be specific to locations.  I do agree though, that we should have a cached function which returns the `last_modified` for all of these, then this function can be rewritten to use that lookup.